crypto_api/src/main/kotlin/org/kethereum/cryptoapi/ec/KeyPairGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package org.kethereum.cryptoapi.ec
+
+import java.math.BigInteger
+
+typealias PrivateAndPublicKey = Pair<BigInteger, BigInteger>
```

Comment:
Let's use ECKeyPair defined in `org.kethereum.crypto.model` here

Reply:
I left it out to avoid circular dependencies between the modules, but I will figure something out :)